cmd/snap/cmd_aliases.go

```diff
@@ -45,8 +45,8 @@ $ snap aliases <snap>
 Lists only the aliases defined by the specified snap.
 
 An alias noted as undefined means it was explicitly enabled or disabled but is
-not defined in the current revision of the snap; possibly temporarely (e.g
-because of a revert), if not this can be cleared with snap alias --reset.
+not defined in the current revision of the snap, possibly temporarily (e.g.
+because of a revert). In any case you can clear this with 'snap alias --reset'.
```

Comment:
It feels better to preserve the third-person on documentation:
```
... because of a revert). This may be cleared up with "snap alias --reset".
```
Also, what's our pattern for commands? Double quotes or single quotes?  I vaguely recall double quotes in some error messages saying --help or similar.

Reply:
We have an open bug about moving to single quotes for that.